weedcoco/deepweeds_to_json/deepweeds_imageinfo.json

```diff
@@ -262644,61 +262644,70 @@
             "id": 0,
             "role": "weed",
```

Comment:
Are we retaining the seperate "role" and "species" fields? Or would it be better to ditch them in favour of the catch-all "name" field?

Reply:
I don't mind. This PR is ambivalent to that decision. We could make that decision and update this PR, or do so after this PR is merged.